### PR TITLE
*: update kube vendor to v1.7.4

### PR DESCRIPTION
**Pull request by runcom**
Signed-off-by: Antonio Murdaca <runcom@redhat.com>

### CODE REVIEW

**Comment by runcom** [comment]
@mrunalp @rhatdan PTAL and please avoid merging stuff which conflicts with vendor file :stuck_out_tongue:

File: test/network.bats [hunk 1]
```diff
@@ -34,20 +46,29 @@ load helpers
...
 	echo "$output"
 	[ "$status" -eq 0 ]
 	pod2_id="$output"
+	run crioctl ctr create --config "$TESTDATA"/container_redis.json --pod "$pod2_id"
+	echo "$output"
+	[ "$status" -eq 0  ]
+	ctr2_id="$output"
 
-	ping_pod_from_pod $pod1_id $pod2_id
+	ping_pod_from_pod $ctr1_id $ctr2_id
```
Comment (TomSweeneyRedHat) [inline test/network.bats]: Any value to keeping the test for pod1 to pod2 in play and adding the ctr1 to ctr2 test in addition?

**Comment by TomSweeneyRedHat** [comment]
Do we now have a golang 1.8+ requirement for CRI-O?  If so, I think that should be mentioned somewhere in the README.md

Otherwise another small question and LGTM

**Comment by mrunalp** [comment]
[retest]

**Comment by runcom** [comment]
[retest]

**Comment by runcom** [comment]
[retest]

**Comment by runcom** [comment]
no clue why Travis isn't happy with this.

**Comment by runcom** [comment]
[retest]

**Comment by runcom** [comment]
[retest]

**Comment by runcom** [comment]
`make lint` passes on my laptop in less than 1 minute, not sure why it randomly fails on Travis

**Comment by runcom** [comment]
awesome, CI passed :)

**Comment by mrunalp** [comment]
The one failure is for the patch that we carry? We should probably add it to our branch for 1.7 tests.

**Comment by runcom** [comment]
I'll add the patch yeah

**Comment by mrunalp** [comment]
There is a script issue in travis :/

**Comment by runcom** [comment]
> The one failure is for the patch that we carry? We should probably add it to our branch for 1.7 tests.

patch added, now CI should test with kube 1.7.4 + your log patch :)

**Comment by runcom** [comment]
These above are old CI runs, no worry :)

**Comment by mrunalp** [comment]
[retest]

**Comment by mrunalp** [comment]
LGTM

**Comment by rhatdan** [comment]
Does this mean we now support Kubernetes 1.7 and not 1.6?

**Comment by runcom** [comment]
@rhatdan yes, target is now 1.7

**Comment by runcom** [comment]
Old 1.6 code is in `release-1.6` branch for anyone using it.

**Comment by rhatdan** [comment]
But we are still going to shop cri-o.1.0 and then for kubernetes 1.8 we ship cri-o-1.8?

**Comment by runcom** [comment]
Yup (replied on email as well)

**Pull request merged**